### PR TITLE
Fix the background not rendering on the second frame

### PR DESCRIPTION
**Pull request by samcake**
This was simply due to a bug in the GLBackend which was actually expected to happen...
  // TODO: we will probably need to also check the write mask of stencil like we do
 // for depth buffer, but as would say a famous Fez owner "We'll cross that bridge when we come to it"

I guess we crossed that bridge today and fixed the stencilWriteMask not being set to xFF and preventing to write the Stencil buffer during the glClear call.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1873,8 +1873,8 @@ void Application::initializeGL() {
         isDeferred = false;
     }
 
-    _renderEngine->addJob<MainRenderTask>("MainFrame", cullFunctor, isDeferred);
     _renderEngine->addJob<SecondaryCameraRenderTask>("SecondaryCameraFrame", cullFunctor);
+    _renderEngine->addJob<MainRenderTask>("MainFrame", cullFunctor, isDeferred);
```
Comment (zfox23) [inline interface/src/Application.cpp]: Did you mean to flip the ordering in this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/636/HighFidelity-Beta-PR10673-f973f7b5e34c4e77f994ee225078e0f238e265aa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/636/HighFidelity-Beta-PR10673-f973f7b5e34c4e77f994ee225078e0f238e265aa.exe

**Comment by ZappoMan** [comment]
Bridges are overrated. Sometimes getting your feet wet is worth it.

File: interface/src/Application.cpp (hunk 1 above)
Comment (samcake) [inline interface/src/Application.cpp]: yep that too

Comment (zfox23) [inline interface/src/Application.cpp]: Why is that necessary?

**Comment by zfox23** [comment]
Yay, works! I'll resolve the conflicts and merge into `spectator-camera`.
